clients/TypeScript/packages/client/src/OgmiosHealth.ts

```diff
@@ -0,0 +1,48 @@
+import fetch from 'cross-fetch'
+import {
+  Connection,
+  ConnectionConfig,
+  createConnectionObject,
+  isConnectionObject
+} from './Connection'
+import { Tip } from '@cardano-ogmios/schema'
+
+export interface OgmiosHealth {
+  currentEra: 'Alonzo' | 'Byron' | 'Mary' | 'Shelley'
+  lastKnownTip: Tip,
+  lastTipUpdate: string | null,
+  metrics: {
+    runtimeStats: {
+      gcCpuTime: number,
+      cpuTime: number,
+      maxHeapSize: number,
+      currentHeapSize: number
+    },
```

Comment:
The runtime stats are only available if run with the right runtime flags (which is the case by default). I don't think there's a particular issue here though, but it could lead to confusing error messages.

Reply:
I'll update the type def

Reply:
Is the option for deactivating `+RTS -T -RTS`? The [docs](https://ogmios.dev/getting-started/monitoring/) say it's enabled using `+T` but is enabled by default, but the example uses `-T` to "run Ogmios with runtime stats **activated**." Is this a mistake?

I tried to adding it to [cardano-node-ogmios](https://github.com/CardanoSolutions/ogmios/blob/master/scripts/cardano-node-ogmios.sh#L29-L32) :

``` 
ogmios \
  --host 0.0.0.0\
  --node-config /config/cardano-node/config.json \
  --node-socket /ipc/node.socket \
  +RTS -T -RTS
  ```
  ... but am still seeing the `runtimeStats`

Reply:
+RTS -T -RTS is for activating the runtime stats. Passing +RTS -RTS should deactivate them.

Reply:
I'm still seeing runtime stats with this command:
```
ogmios \
  --host 0.0.0.0\
  --node-config /config/cardano-node/config.json \
  --node-socket /ipc/node.socket \
  +RTS -RTS
```
``` json
{
   "severity":"Info",
   "timestamp":"2021-07-04T02:08:11.003115346Z",
   "thread":"17",
   "message":{
      "tag":"OgmiosHealth",
      "healthCheck":{
         "tag":"HealthTick",
         "status":{
            "startTime":"2021-07-04T02:06:40.667217866Z",
            "lastKnownTip":{
               "headerHash":"306480300948caaa509e4b2dd44a80818ecddc1bccd41513eea9a3af713d4bd2",
               "slotNo":51655,
               "blockNo":50625
            },
            "lastTipUpdate":"2021-07-04T02:08:11.002889343Z",
            "networkSynchronization":0.01683,
            "currentEra":"Byron",
            "metrics":{
               "activeConnections":0,
               "totalConnections":0,
               "totalUnrouted":0,
               "sessionDurations":{
                  "mean":0,
                  "min":0,
                  "max":0
               },
               "runtimeStats":{
                  "currentHeapSize":338,
                  "gcCpuTime":113789651,
                  "cpuTime":4979847237,
                  "maxHeapSize":339
               },
               "totalMessages":0
            }
         }
      }
   },
   "version":"v4.0.0-??-????????"
}
```
I'll just make `runtimeStats` an optional property anyway if that's the intent.